pypiserver/_app.py

```diff
@@ -373,6 +373,8 @@ def server_static(filename):
 
 
 @app.route("/:project/json")
+@app.route("/pypi/:project/json")
+@app.route("/simple/:project/json")
```

Comment:
@matteius, oh, right! could you elaborate a little on why these routes were added? :) I'm not sure I recognize it fully, in case this is not strictly critical for this feature, do you mind if we add the routes in a separate PR? 🤔